### ISSUE TITLE
The Meilix road block project

### ISSUE THREAD

**Issue by meets2tarun** [fossasia/labs.fossasia.org#277]
Maturate Meilix into a modern usable desktop environment ready to deploy. Minimum achievement would be full 64 bit transition and solving other major issues. Desirable knowledge is experience with Debian packaging and the system boot process, It would be great if you have LPIC certification or equivalent skill level, experience with LinuxFromScratch and shell scripting. If you show the ability to follow up on requests and solve difficult issues in a team you would be a good match to the project.

- [ ]  64 Bit https://github.com/fossasia/meilix/issues/24
- [ ]   64 Bit app support https://github.com/fossasia/meilix/issues/201
- [ ]  Killing Desktop notifications https://github.com/fossasia/meilix/issues/133
- [X] Add harddisc install https://github.com/fossasia/meilix/issues/144 
- [ ]  Minor issues e.g. https://github.com/fossasia/meilix/issues/200
    
and others issues in https://github.com/fossasia/meilix

**Comment by ssirowa** [comment]
@meets2tarun, I'm a Computer Science and Engineering Student at Indian Institute of Technology. I would like to work on this during GSoC. Can you help me get started. I can learn all the required stuff mentioned above before the coding officially starts.

**Comment by meets2tarun** [comment]
Yes thanks @ssirowa, Welcome to the project. 
The project is basically to get the 64 bit of Meilix which is now 32 bit version.

For Getting Started, go through readme and try to deploy meilix into your system locally and test out the feature of the ISO.

**Comment by tabesin** [comment]
Here are the guides for GSOC proposals at fossasia https://docs.google.com/document/d/1E4kNgrOSw64R2IAG45JSIgQL9Yxia-mG2dqlRcrUo-U/edit#

**Comment by hims1911** [comment]
Hello , I'm CSE student and I want to apply for GSOC, can anyone tell me what are the skills required in these project ? Knowledge of Debian packaging and error handling is sufficient or I have to learn some other skills regarding these?

**Comment by tabesin** [comment]
Hi! Look at the files in the meilix repo https://github.com/fossasia/meilix, in particular build.sh and chroot.sh. Everytime we submit a PR a new ISO gets generated on the travis ci server with these scripts. You find the LIVE-CD ISO image under "releases". 

You could also fork the Meilix repository (master branch) and make a local checkout and run the build process on your machine.

Meilix is based on the original build script for Lubuntu from 2012. We modernised it a bit last year but what is still missing is 64Bit images. For that we need to build a 64bit boot lzma kernel and there may be also some differences in the startup process process.

See the bug reports 
https://github.com/fossasia/meilix/issues/24

Basically what you need to find out is how Debian and other Debian based distributions generate their 84Bit ISO Live CDs and research what we are missing and also look at the incomplete attempts of all the other project members to solve the issue. Basically no one so far took the time to look systematically into the issue.
